src/color-slider/color-slider.js

```diff
@@ -260,7 +287,7 @@ defineFormAssociated(ColorSlider, {
 	getSource: el => el._el.slider,
 	role: "slider",
 	valueProp: "value",
-	changeEvent: "input",
+	changeEvent: "change",
```

Comment:
Is this an additional event to the ones we already have (`input` and `colorchange`)?

Reply:
Neither would be correct actually. This is the event that updates this control's form associated value, which is `this.value`. I think I need to convert `Prop` to using an event instead of the callback (which I've been wanting to do anyway), then `defineformAssociated()` can just listen to prop changes.

Reply:
Please correct me if I'm wrong. Having this, let me use `<color-slider>` in my Mavo app, and all I need to do to make it work is to teach Mavo that this element should be handled as a form control, like so

```html
<color-slider property="color" mv-edit-as="formNumber" space="oklch" stops="gold, darkcyan, indigo"></color-slider>
```

And this should work, right?

Reply:
Can't remember what `formNumber` is, you'd need to pick the one that `<input type=range>` is under.

Reply:
Awesome! Thanks.